pkg/apis/serving/v1alpha1/knativeserving_types.go

```diff
@@ -28,6 +28,13 @@ const (
 	DeploymentsAvailable apis.ConditionType = "DeploymentsAvailable"
 )
 
+// Registry defines the registry of KnativeServing
+// +k8s:openapi-gen=true
+type Registry struct {
+	DefaultRegistry string `json:"default-registry,omitempty"`
+	DefaultTag      string `json:"default-tag,omitempty"`
+}
```

Comment:
What is the "Default*" prefix meant to imply? Why not just `Name` and `Tag` as the only elements of the `Registry` struct?

Reply:
Per comment https://github.com/knative/serving-operator/pull/20#issuecomment-505933467, it demonstrates the that this is the default registry and tag to use. I am starting to prefer the nested default map.